src/main/java/com/notably/model/ModelManager.java

```diff
@@ -143,4 +155,39 @@ public boolean equals(Object obj) {
                 && filteredPersons.equals(other.filteredPersons);
     }
 
+    //=========== Suggestion Model =============================================================
+    @Override
+    public ObservableList<SuggestionItem> getSuggestions() {
+        return suggestionModel.getSuggestions();
+    }
+
+    @Override
+    public void setSuggestions(List<SuggestionItem> suggestions) {
+        suggestionModel.setSuggestions(suggestions);
+    }
+
+    @Override
+    public Property<Optional<String>> getCommandTextProperty() {
+        return suggestionModel.getCommandTextProperty();
+    }
+
+    @Override
+    public void setCommandInputText(String commandInputText) {
+        suggestionModel.setCommandInputText(commandInputText);
+    }
+
+    @Override
+    public String getCommandInputText() {
+        return suggestionModel.getCommandInputText();
+    }
+
+    @Override
+    public void clearCommandInputText() {
+        suggestionModel.clearCommandInputText();
+    }
```

Comment:
Do remove these methods here too once you removed the `commandInputText` field from #123 !

Reply:
done, thanks bro!